cirrus/google_cloud/manager.py

```diff
@@ -104,6 +104,7 @@ def __init__(self, project_id=None, creds=None, use_default=True):
         )
         creds = creds or config.GOOGLE_APPLICATION_CREDENTIALS
         self.credentials = ServiceAccountCredentials.from_service_account_file(creds)
+        self.open_count = 0
```

Comment:
lets name this _open_count with a leading underscore so people don't try to use this. or __enter_count__, that seems excessive though

Reply:
I would add a quick clarification comment here too. `This allows for safely re-opening the object multiple times and prevents issues with attempting to close more than needed`. something like that.